lib/apiservers/engine/backends/container.go

```diff
@@ -534,9 +534,7 @@ func (c *Container) containerStart(name string, hostConfig *containertypes.HostC
 	if bind {
 		e := c.findPortBoundNetworkEndpoint(hostConfig, endpoints)
 		if err = c.mapPorts(portmap.Map, hostConfig, e); err != nil {
-			err = fmt.Errorf("error mapping ports: %s", err)
-			log.Error(err)
```

Comment:
Hmm wait, we log it when we get the InternalServer error on the other side like the rest of the errors.